packages/ovm/src/contracts/ExecutionManager.sol

```diff
@@ -118,52 +117,37 @@ contract ExecutionManager is FullStateManager {
      *       [callBytes (bytes (variable length))]
      * returndata: [variable-length bytes returned from call]
      */
-    function executeCall() external {
+    function executeTransactionRaw() external {
         uint _timestamp;
         uint _queueOrigin;
-        uint callSize;
-        bytes memory callBytes;
-        bytes32 methodId = ovmCallMethodId;
+        uint _callSize;
+        bytes memory _callBytes;
+        address _ovmEntrypoint;
         assembly {
-            // Revert if we don't have methodId, timestamp, queueOrigin, and ovmEntrypointAddress.
-            if lt(calldatasize, 100) {
-                revert(0,0)
-            }
-
             // populate timestamp and queue origin from calldata
             _timestamp := calldataload(4)
             // skip method ID (bytes4) and timestamp (bytes32)
-            _queueOrigin := calldataload(0x24)
+            _queueOrigin := calldataload(add(4, 32))
```

Comment:
I think we should either be all hex or all decimal. Also, any reason for adding the extra `add` instruction?